packages/fast-components-react-msft/src/metatext/examples.data.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { IComponentFactoryExample } from "@microsoft/fast-development-site-react";
+import Metatext from "./index";
+import {
+    IMetatextHandledProps,
+    IMetatextManagedClasses,
+    IMetatextUnhandledProps,
+    MetatextProps,
+    MetatextTag
+} from "./metatext.props";
+import schema from "./metatext.schema.json";
+import Documentation from "./.tmp/documentation";
```

Comment:
Can you explain what's happening here?  Why is documentation in a `.tmp` folder?

Reply:
I think any generated files that are used for deployment should go in a dist or www folder if one of those is already present.  my preference is `dist` since it's more comment.  Also, prefer to have `www` inside that if there is an accompanying website.  But, this can be a future change and has no bearing on this PR.